invenio_app_ils/ill/search.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""ILL search APIs."""
+
+from invenio_search.api import RecordsSearch
+
+from invenio_app_ils.errors import MissingRequiredParameterError
+
+
+class LibrarySearch(RecordsSearch):
+    """Search for ILL libraries."""
+
+    class Meta:
+        """Search only on libraries index."""
+
+        index = "ill-libraries"
+        doc_types = None
+
+
+class BorrowingRequestsSearch(RecordsSearch):
+    """Search for ILL borrowing requests."""
+
+    class Meta:
+        """Search only on borrowing requests index."""
+
+        index = "ill-borrowing-requests"
```

Comment:
I believe we use underscores to separate words in other schemas but it should be fine either way. The only thing we'll need to do is convert the `RECORDS_REST_SORT_OPTIONS` config to be instantiated with `{}` instead of `dict()` since we can't use dashes in the keys there.